plan/plan.go

```diff
@@ -408,12 +407,14 @@ func resolveExtraVars(vars ...map[string]string) map[string]string {
 	return resolved
 }
 
-func resolveAccounts(accounts map[string]v2.Account) map[string]json.Number {
-	a := make(map[string]json.Number)
+func resolveAccounts(accounts map[string]v2.Account) map[string]*json.Number {
+	a := make(map[string]*json.Number)
 	for name, account := range accounts {
-		if account.Providers != nil && account.Providers.AWS != nil && account.Providers.AWS.AccountID != nil {
-			a[name] = *account.Providers.AWS.AccountID
+		var acctID *json.Number
+		if account.Providers != nil && account.Providers.AWS != nil {
```

Comment:
Just curious, how come you removed the`account.Providers.AWS.AccountID != nil` conditional?

Reply:
it's ok for it to be nil, we then do a corresponding [nil check](https://github.com/chanzuckerberg/fogg/blob/master/templates/account/fogg.tf.tmpl#L79) in the templates (which we were already doing)